pynetdicom/tests/test_ae.py

```diff
@@ -694,7 +694,7 @@ def test_ae_title_empty_str(self):
     def test_ae_title_invalid_chars(self):
         """ AE should fail if ae_title is not a str """
         with pytest.raises(ValueError):
-            AE(ae_title=b'TEST\ME')
+            AE(ae_title=rb'TEST\ME')
```

Comment:
Fails with Python 2.7

Reply:
I have used br"" to resolve this. I didn't know about this in Python 2 : https://bugs.python.org/issue13748